lib/matplotlib/image.py

```diff
@@ -890,7 +819,7 @@ def __init__(self, ax,
         self.update(kwargs)
         self.set_data(x, y, A)
```

Comment:
Thanks.  This is something that I actually *didn't* change.  I guess I'll update for `super` throughout this file.